Library/Homebrew/cmd/search.rb

```diff
@@ -129,29 +130,21 @@ def search_pull_requests(query, args)
     GitHub.print_pull_requests_matching(query, only)
   end
 
-  def search_names(query, string_or_regex, args)
-    remote_results = search_taps(query, silent: true)
-
-    local_formulae = search_formulae(string_or_regex)
-    remote_formulae = remote_results[:formulae]
-    all_formulae = local_formulae + remote_formulae
-
-    local_casks = search_casks(string_or_regex)
-    remote_casks = remote_results[:casks]
-    all_casks = local_casks + remote_casks
-
```

Comment:
Moved this to its own function so that it can be reused over in the install command.